doc/user/basic.rst

```diff
@@ -180,13 +180,14 @@ Basic Config Commands
    In this example, the precision is set to provide timestamps with
    millisecond accuracy.
 
-.. index:: log commands
-.. clicmd:: log commands
+.. index:: [no] log commands
+.. clicmd:: [no] log commands
 
    This command enables the logging of all commands typed by a user to all
    enabled log destinations. The note that logging includes full command lines,
-   including passwords. Once set, command logging can only be turned off by
-   restarting the daemon.
+   including passwords. If the daemon startup option `--command-log-always`
+   is used to start the daemon then this command is turned on by default
+   and cannot be turned off and the [no] form of the command is dissallowed.
```

Comment:
At this point, I'd be fine with just the command line - but I guess there is a use case for using log commands temporarily, e.g., for debugging.